modules/util/source-formatter/src/test/java/com/liferay/source/formatter/processor/UpgradeCatchAllCheckTest.java

```diff
@@ -80,30 +80,48 @@ public static Iterable<Object[]> data() throws Exception {
 					issueKey,
 					(key, value) -> {
 						if (value == null) {
-							value = new HashSet<>();
+							value = JSONUtil.put("fileTypes", new HashSet<>());
 						}
 
-						value.add(fileType);
+						Set<String> issueFileTypes = (Set<String>)value.get(
```

Comment:
Hi @nicolas-sss , can we rename `fieldTypes` to `validExtensions` on the line above the `for` ? this way `issueFileTypes` could be renamed to `fileTypes`, I think this would be more consistent.